Exception when deleting Data Flows

When deleting a data model with attached data flows, the following exception occurs:

```
Caused by: groovy.lang.MissingMethodException: No signature of method: static uk.ac.ox.softeng.maurodatamapper.util.Utils.executeInBatches() is applicable for argument types: (SingletonSet, uk.ac.ox.softeng.maurodatamapper.dataflow.DataFlowService$__tt__dataFlowService_deleteAllByModelIds_closure29) values: [[e7f146dc-e9a3-4aad-a3c2-c21f614d2b35], uk.ac.ox.softeng.maurodatamapper.dataflow.DataFlowService$__tt__dataFlowService_deleteAllByModelIds_closure29@6cdab271]
Possible solutions: executeInBatches(java.util.List, groovy.lang.Closure)
	at uk.ac.ox.softeng.maurodatamapper.dataflow.DataFlowService.$tt__dataFlowService_deleteAllByModelIds(DataFlowService.groovy:109)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at grails.gorm.transactions.GrailsTransactionTemplate$2.doInTransaction(GrailsTransactionTemplate.groovy:94)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:140)
	at grails.gorm.transactions.GrailsTransactionTemplate.execute(GrailsTransactionTemplate.groovy:91)
	at uk.ac.ox.softeng.maurodatamapper.datamodel.DataModelService$__tt__dataModelService_deleteModelsAndContent_closure126.doCall(DataModelService.groovy:469)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at uk.ac.ox.softeng.maurodatamapper.datamodel.DataModelService.$tt__dataModelService_deleteModelsAndContent(DataModelService.groovy:464)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at grails.gorm.transactions.GrailsTransactionTemplate$2.doInTransaction(GrailsTransactionTemplate.groovy:94)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:140)
	at grails.gorm.transactions.GrailsTransactionTemplate.execute(GrailsTransactionTemplate.groovy:91)
	at uk.ac.ox.softeng.maurodatamapper.core.model.ModelService.deleteModelAndContent(ModelService.groovy:196)
	at uk.ac.ox.softeng.maurodatamapper.datamodel.DataModelService.$tt__dataModelService_delete(DataModelService.groovy:250)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at grails.gorm.transactions.GrailsTransactionTemplate$2.doInTransaction(GrailsTransactionTemplate.groovy:94)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:140)
	at grails.gorm.transactions.GrailsTransactionTemplate.execute(GrailsTransactionTemplate.groovy:91)
	at uk.ac.ox.softeng.maurodatamapper.core.model.ModelService.permanentDeleteModel(ModelService.groovy:222)
	at uk.ac.ox.softeng.maurodatamapper.core.controller.ModelController.$tt__modelController_delete(ModelController.groovy:153)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at grails.gorm.transactions.GrailsTransactionTemplate$2.doInTransaction(GrailsTransactionTemplate.groovy:94)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:140)
	at grails.gorm.transactions.GrailsTransactionTemplate.execute(GrailsTransactionTemplate.groovy:91)
```

--- a/mdm-plugin-dataflow/grails-app/services/uk/ac/ox/softeng/maurodatamapper/dataflow/DataFlowService.groovy
+++ b/mdm-plugin-dataflow/grails-app/services/uk/ac/ox/softeng/maurodatamapper/dataflow/DataFlowService.groovy
@@ -106,7 +106,7 @@ class DataFlowService extends ModelItemService<DataFlow> {
 
             log.trace('Removing {} DataFlows', dataFlowIds.size())
 
-            Utils.executeInBatches(modelIds, { ids ->
+            Utils.executeInBatches(modelIds as List, { ids ->
                 sessionFactory.currentSession
                         .createSQLQuery('DELETE FROM dataflow.data_flow WHERE source_id in :ids OR target_id in :ids')
                         .setParameter('ids', ids)

